src/utils/git.ts

```diff
@@ -13,7 +13,7 @@ export async function getCurrentCommitInfo(): Promise<{
   const branchStr = await shell("git branch --show-current");
   const timestamp = parseInt(timestampStr, 10);
 
-  if (!timestamp || isNaN(timestamp)) {
+  if (!timestamp || Number.isNaN(timestamp)) {
```

Comment:
@matthewkeil This usage is different than the `Number` namespace rule. This is safety fix as the behavior of global `isNaN` and `Number.isNaN` is different. Global instance coerce the input to number before checking it, later one check what the given input is.